ajax/libs/tslib/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tslib",
+  "filename": "tslib.min.js",
+  "author": "Microsoft Corp.",
+  "homepage": "http://typescriptlang.org/",
```

Comment:
hi, the link http://typescriptlang.org/ does not work with https for me, the link with www works fine with https, could we have that merged?